parm/ioda/bufr2ioda/bufr2ioda_acft_profiles_prepbufr.json

```diff
@@ -6,7 +6,7 @@
   "cycle_type"       : "{{ RUN }}",
   "cycle_datetime"   : "{{ current_cycle | to_YMDH }}",
   "dump_directory"   : "{{ DMPDIR }}",
-  "ioda_directory"   : "{{ COM_OBS }}",
+  "ioda_directory"   : "{{ COMIN_OBS }}",
```

Comment:
Pedantic: but I suspect most of these should be COMOUT_OBS if they are generating obs to put in COM